core/templates/components/forms/custom-forms-directives/edit-thumbnail-modal.component.html

```diff
@@ -26,12 +26,10 @@ <h3>Upload Thumbnail</h3>
       <div class="oppia-form-error" *ngIf="invalidFilenameWarningIsShown">
         Please upload a valid filename image i.e; The svg file must not start with a dot or have '/' or 2 consecutive dots.
       </div>
-      <div class="oppia-form-error" *ngIf="invalidTagsAndAttributes?.tags.length">
-        Unsupported SVG tags found: {{ invalidTagsAndAttributes.tags }}
-      </div>
-      <div class="oppia-form-error" *ngIf="invalidTagsAndAttributes?.attrs.length">
-        Unsupported SVG attributes found: {{ invalidTagsAndAttributes.attrs }}
-      </div>
+      <div *ngIf="invalidTagsAndAttributes.tags.length || invalidTagsAndAttributes.attrs.length"
+           class="oppia-form-error oppia-unsupported-svg-tag-attribute-error"
+           [innerHTML]="'I18N_INVALID_TAGS_AND_ATTRIBUTES_ALERT' | translate: { issueURL: svgSanitizerService.getIssueURL(invalidTagsAndAttributes) }"
+      ></div>
```

Comment:
I think it makes more sense for the ">" to be at the end of the previous line?

Reply:
Done!